basic_test.py

```
@@ -0,0 +1,49 @@
+        # Test if we can import the main module
+        sys.path.insert(0, os.path.dirname(__file__))
+
+        # Try importing version
+        try:
+            with open(os.path.join(src_path, 'version.py'), 'r') as f:
+                version_content = f.read()
+                if '__version__' in version_content:
+                    print("✅ Version file exists")
+                else:
+                    print("⚠️  Version file found but no version info")
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)